app/models/unit.rb

```diff
@@ -392,42 +408,25 @@ def employ_staff(user, role)
   end
 
   # Adds a user to this project.
-  def enrol_student(user, campus, tutorial = nil)
-    tutorial_id = if tutorial.is_a?(Tutorial)
-                    tutorial.id
-                  else
-                    tutorial
-                  end
-
+  def enrol_student(user, campus)
```

Comment:
Remove `tutorial` from method call (line 717)

Reply:
Yes, I think we will have to now. Where was this used?